tools/ci-resources/tls-stub/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+[package]
+name = "stub"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+aws-config = {path = "../../../aws/sdk/build/aws-sdk/sdk/aws-config", features = ["client-hyper"] }
+aws-credential-types = { path = "../../../aws/sdk/build/aws-sdk/sdk/aws-credential-types", features = ["hardcoded-credentials"] }
+aws-sdk-sts = { path = "../../../aws/sdk/build/aws-sdk/sdk/sts" }
```

Comment:
can you add docs here explaining what this package is (or add a readme?)

Reply:
also a `publish = false` for good measure

Reply:
Done, thanks